terraform/services.tf

```diff
@@ -0,0 +1,35 @@
+resource "google_project_service" "compute-api" {
```

Comment:
I just found out that service accounts (service_accounts.tf) depend on the iam.googleapis.com API.